ElectronNET.API/ElectronNET.API.csproj

```diff
@@ -17,32 +17,38 @@ This package contains the API to access the "native" electron API.</Description>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <PackageTags>electron aspnetcore</PackageTags>
     <PackageReleaseNotes>Changelog: https://github.com/ElectronNET/Electron.NET/blob/master/Changelog.md</PackageReleaseNotes>
-    <PackageIconUrl>https://raw.githubusercontent.com/ElectronNET/Electron.NET/master/assets/images/electron.net-logo-square.png</PackageIconUrl>
-    <Version>1.0.0.0</Version>
+    <PackageIcon>PackageIcon.png</PackageIcon>
+    <Version>99.0.0.0</Version>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
Better replace lines which define `DocumentationFile` this one big switch with just 
```
<GenerateDocumentationFile>true</GenerateDocumentationFile>
```
This will have same effect but without need to generate all permutations. But this one change can be made separately from this PR

Reply:
Could you point me to a documentation? I just found some links that said that this controls the XML documentation behavior. Those lines are for the NuGet package icon or do I miss something?

Reply:
I cannot say anything about documentation, since I found that long time ago when lurking around MS projects or people who closely monitor .NET Core development.

Essentially this switch produce XML files in the default location, so instead of duplication configuration and paths everywhere you just include that property in the project file. You can see in the #323 how exactly it simplifies project file.

Here the links which you can find interesting:
https://github.com/dotnet/sdk/blob/b28a8efbfb30d84019351abb3aa9a2b15e008662/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets#L178

https://stackoverflow.com/questions/47115877/how-to-generate-xml-documentation-for-csproj-with-multiple-targets